xtas/tasks/_alpino.py

```diff
@@ -33,7 +33,7 @@
 
 log = logging.getLogger(__name__)
 
-CMD_PARSE = ["bin/Alpino", "end_hook=dependencies", "-parse"]
```

Comment:
The older version linked above seems to be gone, and the new ones do run, but give different results, which I'm not sure are usable for us.